features/step_definitions/web_steps.rb

```diff
@@ -313,3 +313,7 @@ def verified_visit(path)
   expect(page).to have_css("##{editor_id}", visible: false)
   execute_script("tinyMCE.getInstanceById('#{editor_id}').setContent('#{html}');")
 end
+
+When /^I reload the page$/ do
+  page.evaluate_script 'window.location.reload()'
```

Comment:
Wow, I love this compared to 

`visit [ current_path, page.driver.request.env['QUERY_STRING'] ].reject(&:blank?).join('?')`

👍